mailpoet/assets/js/src/notices/email_volume_limit_notice.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactStringReplace from 'react-string-replace';
+import ReactHtmlParser from 'react-html-parser';
+import MailPoet from 'mailpoet';
+import Notice from 'notices/notice';
+
+function EmailVolumeLimitNotice(): JSX.Element {
+  if (!MailPoet.emailVolumeLimitReached) return null;
+
+  const title = MailPoet.I18n.t('emailVolumeLimitNoticeTitle')
+    .replace('[emailVolumeLimit]', MailPoet.emailVolumeLimit);
+  const youReachedEmailVolumeLimit = MailPoet.I18n.t('youReachedEmailVolumeLimit')
+    .replace('[emailVolumeLimit]', MailPoet.emailVolumeLimit);
+  const upgradeLink = `https://account.mailpoet.com/orders/upgrade/${MailPoet.pluginPartialKey}`;
```

Comment:
We have [a factory for building such links](https://github.com/mailpoet/mailpoet/blob/3819dd774e53505616b548236b127d51db7230b4/mailpoet/assets/js/src/mailpoet_com_url_factory.js). It is available on [MailPoet.MailPoetComUrlFactory](https://github.com/mailpoet/mailpoet/blob/18b7139bec772fd4d4deef53fab7b87728062555/mailpoet/assets/js/src/mailpoet.ts#L17).
I think we should create the link using the factory since it adds a referral id that we want to have for every link that goes to the shop.

Saying that realized that we probably don't add the referral id in t[he messages we create in the MSS error mapper](https://github.com/mailpoet/mailpoet/blob/3819dd774e53505616b548236b127d51db7230b4/mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php#L104-L151). So I guess we should add it there as well. Probably in a different task. Or maybe we could as @NeosinneR if the referral id is still needed. I haven't heard about the referrals for some time so maybe we could think of removing this functionality.